index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Survey Form</title>
+</head>
+
+<body>
+  <header>
+    <h1>Survey Form</h1>
+    <p>Please enjoy this sick survey form</p>
+    <h6>(I worked really hard on it)</h6>
+  </header>
+  <main>
+    <form action="" method="GET">
+      <section>
+        <label for="name">Name</label>
+        <input type="text" name="name" required><br>
+      </section>
```

Comment:
Organizing the inputs and their labels into sections is a good instinct, however in this case forms have a dedicated and semantic element for this purpose, the `<fieldset>` element.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

I would recomment switching all sections to fieldsets.